themes/qgis-theme/layout.html

```diff
@@ -24,8 +24,19 @@
   {%- set titlesuffix = "" %}
 {%- endif %}
 
+{%- macro outdated_block() %}
+    <div class="row outdated">
+      This documentation is for a QGIS version which has reached end of life. Visit the <a href="https://docs.qgis.org/latest/">latest version</a> instead.
```

Comment:
Don't you keep the language anymore?
```suggestion
      This documentation is for a QGIS version which has reached end of life. Visit the <a href="https://docs.qgis.org/latest/{{ language }}/">latest version</a> instead.
```